packages/formdata-event/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
```

Comment:
These formdata-event files shouldn't be here, right?

Reply:
Nevermind, just realized this is a huge merge too!

Reply:
Yeah, they're already in master but they show up in this diff since we haven't merged master into html-imports since that work was started.